boltdesignsystem/bolt
# DS-408: Convert yeoman to plop alternative

## Jira

https://pegadigitalit.atlassian.net/browse/DS-408

## Summary

Rewriting the component generator from yeoman to plop and adding the element generator.

## Details

The previous yeoman generator is removed. In his place, we added a plop generator that is easier to use and more developer-friendly. Except we rewrite the component generator we add elements generator.

## How to test

1. Pull the branch locally and run `yarn setup`.
2. Run `yarn cc` to generate component and check if all the files are generated correctly.
3. Run `yarn jest packages/generators/bolt-generator/__tests__/component/bolt-generator.js` and check if all tests passed.
4. Run `yarn ce` to generate element and check if all the files are generated correctly.
5. Run `yarn jest packages/generators/bolt-generator/__tests__/element/bolt-generator.js` and check if all tests passed.

We need only two acceptations but I want you all to check this code.
